test/test_other.py

```diff
@@ -12326,6 +12326,21 @@ def test_build_fetch_examples(self):
     self.build(test_file('fetch/example_waitable_xhr_to_memory.c'))
     self.build(test_file('fetch/example_xhr_progress.c'))
 
+  def test_fetch_init_node(self):
+    # Make sure that `Fetch` initialises correctly under Node where
+    # IndexedDB isn't available.
+    self.set_setting('FETCH')
+    self.set_setting('EXPORTED_RUNTIME_METHODS', ['Fetch'])
+    create_file('src.c', r'''
+#include <stdio.h>
+int main() {
+  puts("ok");
+}
+''')
+    self.run_process([EMCC, 'src.c', '-sFETCH', '-sEXPORTED_RUNTIME_METHODS=["Fetch"]'])
+    ret = self.run_process(config.NODE_JS + ['a.out.js'], stdout=PIPE).stdout
+    self.assertContained('ok', ret)
```

Comment:
In that case is the Fetch module still included after https://github.com/emscripten-core/emscripten/pull/18329/commits/0991eb1bbcc910c7d91e1ee324296877332b9105 ?

Reply:
You can either do:

```
self.set_setting('FETCH')		
self.set_setting('EXPORTED_RUNTIME_METHODS', ['Fetch'])
self.runf('src.c', 'ok')
```

or 

```
self.runf('src.c', 'ok', emcc_args=['-sFETCH', '-sEXPORTED_RUNTIME_METHODS=Fetch'])
```

I lean towards the latter since it keep the diff smaller in this case.